frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -98,38 +99,59 @@ function createFormGroup(controlId, formLabel, inputType, placeholder, ref) {
   }
 
   return (
-    <Form.Group controlId={controlId}>
+    <>
       <Form.Label>{formLabel}</Form.Label>
-        {formControl}
-    </Form.Group>
+      <Form.Group controlId={controlId}>
+          {formControl}
+        {/* Temporary instructions until fix Issue #52 */}
+      </Form.Group>
+    </>
   )
 }
 
 /**
- * Component corresponding to add trips modal.
+ * Component corresponding to the save trips modal.
+ *
+ * This component "acts" as a parent of the (non-existant) AddTripModal and
+ * EditTripModal components. The only differences in the implementation between
+ * the two fake components are dervied from the props `title`, `tripid`, and
+ * `placeholderObj` (see below).
  *
  * @param {Object} props These are the props for this component:
  * - db: Firestore database instance.
  * - show: Boolean that determines if the add trips modal should be displayed.
  * - handleClose: The function that handles closing the add trips modal.
  * - refreshTripsContainer: Function that handles refreshing the TripsContainer
  *        component upon trip creation (Remove when fix Issue #62).
+ * - title: The title of the modal.
```

Comment:
What would this title be? Is it something that would be required to be passed in? i.e. if the title is "edit trip" vs "create new trip", can you check if tripId==null and set the title that way? 
This is fine too I guess, it's just easier to follow if there are fewer black boxes.

Reply:
Yes this one I could pretty easily do inside of save-trip-modal so I can remove this prop.